lib/travis/build/script/smalltalk.rb

```diff
@@ -0,0 +1,26 @@
+module Travis
+  module Build
+    class Script
+      class Smalltalk < Script
+        DEFAULTS = {}
+
+        def setup
```

Comment:
`setup` runs after `sudo` is disabled, so this won't work. You need to do everything you need with `sudo` in `configure`. See http://docs.travis-ci.com//user/languages/community-supported-languages/#Code